tools/account_info.py

```diff
@@ -0,0 +1,46 @@
+import httpx
+import json
+from constants import GENESIS_ACCOUNT, TEST_ACCOUNT, JSON_RPC_URL
+from time import sleep
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests.account_info import AccountInfo
+from xrpl.models.response import ResponseStatus
+
+# Creates client talking to a node in our testnet.
+client = JsonRpcClient(JSON_RPC_URL)
+
```

Comment:
```suggestion
```

Reply:
This generates warning in IDE. (Python has this strange idea that empty line matters).